lib/sinon-match-integration.test.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var assert = require("@sinonjs/referee").assert;
+var samsam = require("./samsam");
+var match = samsam.match;
+
+describe("sinon-match-integration", function() {
+    context("samsam.match", function() {
+        it("returns true if matching custom matcher", function() {
+            var matchAlways = samsam.createMatcher(function() {
+                return true;
+            });
+            var checkMatch = match(42, matchAlways);
+            assert.isTrue(checkMatch);
```

Comment:
Could we make this look like every other tests for the sake of consistency?

```js
assert.isTrue(samsam.match(42, matchAlways));
```

Reply:
Done